decidim-core/config/locales/en.yml

```diff
@@ -276,6 +276,7 @@ en:
   errors:
     messages:
       content_type_whitelist_error: The file type is not valid
+      cycle_detected: a scope descendant can't be its parent
```

Comment:
Yes, I'm afraid that the problem is the English, not the idea. :see_no_evil: So, I will change the text. 
Regarding the second question, this message is used on model validation, but I do not believe that scenario could happen using the current admin UI.